### PR TITLE
Course4 - Marathi tutorial

### PR DESCRIPTION
**Pull request by AryavT**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
python tutorial in Python
### Why is this valuable (or not)?
it will teach people who speak Marathi how to code in python
### How do we know it's really free?
its on YouTube 
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -274,6 +274,7 @@ Online School
 * [Python All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVCH8fRfZ8sUuKEQQYIFMjtJ) - Stack Learner
 * [Python Bangla Tutorial 2023](https://www.youtube.com/playlist?list=PLNMnAEqLBwmpR8JDBOEl0jrzmH1vPnO7v) - Hablu Programmer
 * [Python Bangla Tutorials for Beginners](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rz5XqMsTk41_j15_6682BN) - Anisul Islam
+* [Python Bangla Tutorial & Data Science Full Course || Full Guidelines for Python, ML, AI, DL & Django](https://www.youtube.com/watch?v=A3FKSsYy0Zg&list=PLKdU0fuY4OFcot0zyVbM1-zKf_eCUK4zQ) - STUDY MART
```
Comment (eshellman) [inline courses/free-courses-bn.md:277]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
https://www.youtube.com/playlist?list=PLKdU0fuY4OFcot0zyVbM1-zKf_eCUK4zQ

escape the `|` or replace with `|`

Comment (eshellman) [reply courses/free-courses-bn.md:277]: no ALL CAPS

File: courses/free-courses-mr.md [hunk 2]
```diff
@@ -83,3 +83,6 @@
 * [Python in Marathi](https://youtube.com/playlist?list=PL9D-kb1y7d4cL3xI0Wk1krRjjiPE4IPUd) - MITU Skillologies
 * [Python Programming in Marathi (मराठी)](https://youtube.com/playlist?list=PLI1D7QZwksP64N_zkmXxr9DAzLy9mJClY) - I.T. मंडळ
 * [Python Tutorial for Beginners In Marathi {आता आपल्या भाषेत कोडिंग } From Basic TO Advance](https://youtube.com/playlist?list=PLFwH5aoadVcnfGG9WtTd-4qYO9gzk773P) - Code Marathi
+* [Python Tutorial For Beginners In Marathi Full 17 Hours Course (With Practice Sets) 🔥](https://www.youtube.com/watch?v=gJb6S8AmlYA) - Code Marathi
```
Comment (eshellman) [inline courses/free-courses-mr.md:86]: no emoji

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-bn.md (hunk 1 above)
Comment (AryavT) [reply courses/free-courses-bn.md]: finished

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-bn.md [hunk 3]
```diff
@@ -273,6 +273,7 @@ Online School
 * [Bangla Python Tutorial for Beginners](https://www.youtube.com/playlist?list=PLlBKlxyCgmsCYJLq9qc5QzaU-oBFJN79B) - Niamul Hasan (StartBit)
 * [Python All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVCH8fRfZ8sUuKEQQYIFMjtJ) - Stack Learner
 * [Python Bangla Tutorial 2023](https://www.youtube.com/playlist?list=PLNMnAEqLBwmpR8JDBOEl0jrzmH1vPnO7v) - Hablu Programmer
+*  [Python Bangla Tutorial & Data Science Full Course](https://www.youtube.com/playlist?list=PLmZYUigljiyc-tf7oMmM-s832ibhGxTpT) - Study Mart
```
Comment (eshellman) [inline courses/free-courses-bn.md:276]:
```suggestion
* [Python Bangla Tutorial & Data Science Full Course](https://www.youtube.com/playlist?list=PLmZYUigljiyc-tf7oMmM-s832ibhGxTpT) - Study Mart
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**